packages/trader/package.json

```diff
@@ -104,6 +104,7 @@
         "@deriv/quill-icons": "^1.22.8",
         "@types/react-loadable": "^5.5.6",
         "classnames": "^2.2.6",
+        "clsx": "^2.0.0",
```

Comment:
according to the authors of `clsx`, this library is the same as `classnames` but has a smaller bundle size and better performance. Reference: https://www.npmjs.com/package/clsx